### DISCUSSION TITLE
Basic (Ordinary) users with full access to all resources and all operations defined on policies, still cannot access all settings menu

### DISCUSSION THREAD

**Discussion by mchristianto** [open-metadata/OpenMetadata#16522] [Q&A] [answered]
Why basic user even if with policies and role granted to all resources and all operations still cannot access all settings menu, for example Notification Menu, Bot Menu, and Alerts? the error is permission denied

**Comment by Prajwal214** [comment, marked as answer]
Basic users, even with policies and roles granting access to all resources and operations, may still encounter permission issues for certain settings menus like Notification, Bot, and Alerts. This is because some operations and resources in OpenMetadata are restricted to admin users only.

**Reply by mchristianto** [reply]
I got it, thanks for the explanation
